docs/clients.md

```diff
@@ -69,6 +69,43 @@ Therefore these two rules may help when choosing what to use:
 * When opening a client within a service, use local clients.  Usually you'll want to open one client per worker.
 * When opening a client from outside a service, such as from inside an actor or some other general use case, use a future client.
 
+## Failure Management
+ 
+### Client Behavior 
+ 
+If any failure occurs (example: connection closed, external system isn't 
+responding) the response is sent back as a failure and NOT retried.  The 
+default behavior is to continue trying subsequent requests.  If failfast is
+enabled then all subsequent requests that are queued up in the ServiceClient
+are removed.  
```

Comment:
I would be more explicit:

If failfast is enabled then all queued requests in the ServiceClient are immediately failed.